clang/include/clang/Driver/Options.td

```diff
@@ -2074,6 +2074,25 @@ def fbounds_safety_debug_trap_reasons_EQ
     NormalizedValues<["None", "Basic", "Detailed"]>,
     MarshallingInfoEnum<CodeGenOpts<"BoundsSafetyDebugTrapReasons">, "Detailed">;
 
+def fbounds_safety_soft_traps_EQ
+  : Joined<["-"], "fbounds-safety-soft-traps=">, Group<f_Group>,
+    Visibility<[ClangOption, CC1Option]>,
+    HelpText<"Set how trap reasons are emitted. "
+      "`disabled` - No soft traps, i.e. use hard traps (default)."
+      "`call_with_str` -  "
+      "`call_with_code` - ">,
```

Comment:
Can we add to the help text what these do?

Reply:
Fixed